src/test_l1_plat_power.c

```diff
@@ -74,6 +74,30 @@
 static int gTestGroup = 1;
 static int gTestID = 1;
 
+#define PLAT_ASSERT_AUTO_TERM_NUMERICAL(value, comparison){\
+      if(value != comparison){\
+	        UT_LOG("\n In %s Comparison: [%d = %d]\n", __FUNCTION__, value, comparison);\
+		PLAT_TERM();\
```

Comment:
Both the cleanup and termination of the test are done in a single comparison rather than having one for cleanup and one UT core comparison for exit will decrease the modularity.

Reply:
So a function that can automatically call on failure with a callback will be useful. I will add this to the feature list